Examples/Pass/arc.cvf

```diff
@@ -25,13 +21,7 @@ method print() {
   {| arc() * specialViewForFree(f) |}
     // Test for disposal
     if (f == true) {
-      {| error() |}
-        dummy = dummy; // skip;
-      {| error() |}
-    } else {
-      {| arc() |}
-        dummy = dummy; // skip;
-      {| arc() |}
+      {| error() |} ; {| error() |}
```

Comment:
I think we should do it *at some point*, but it's not a priority now. 

Phew, conflict resolved.